### PR TITLE
[AAE-17239] Added a check for uploading file

### LINKED ISSUE #4469: It is not possible to import projects larger than 10mb
Projects larger than 10 MB cannot be imported because of the limit set in the application.yml
It is necessary to establish distinct limits for every endpoint

### PR DESCRIPTION
**Pull request by AkhileshPamidimarthi**
This Pr fixes : https://github.com/Activiti/Activiti/issues/4469

### CODE REVIEW

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java [hunk 1]
```diff
@@ -149,6 +150,9 @@ public void updateModelContent(
         @PathVariable String modelId,
         @RequestPart(UPLOAD_FILE_PARAM_NAME) MultipartFile file
     ) throws IOException {
+        if (file.getSize() > 1024 * 1024 * 10) {
```
Comment (jesty) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java]: please replace the number (1024*1024*10) with a property using @Value annotation

@Value("${activiti.modeling.maxModelFileSize:10485760}")
private long maxModelFileSize;

**Review by jesty — CHANGES_REQUESTED** [review]

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java [hunk 2]
```diff
@@ -77,6 +79,9 @@ public class ModelController implements ModelRestApi {
 
     private final ProjectController projectController;
 
+    @Value("${activiti.modeling.maxModelFileSize:1024102410}")
```
Comment (mavotto) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java]: The value should be 10485760

**Review by mavotto — CHANGES_REQUESTED** [review]

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java (hunk 2 above)
Comment (AkhileshPamidimarthi) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java:82]: there is *10 too right

**Review by jesty — APPROVED** [review]
LGTM

**Review by mavotto — APPROVED** [review]

**Pull request merged**